demo/WebexMeetingWidgetDemo.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Input} from '@momentum-ui/react';
+
+import {WebexMeetingWidget} from '../src';
+
+function WebexMeetingWidgetDemo({token}) {
+  const [destination, setDestination] = useState('');
+  const [displayWidget, setDisplayWidget] = useState(false);
+
+  const displayButtonEnabled = token && destination && !displayWidget;
+
+  return (
+    <div>
+      <Input
+        htmlId="destination"
+        label="Widget Destination (email, room, sip, etc)"
+        name="destination"
+        onChange={(event) => setDestination(event.target.value)}
+        placeholder="Widget Destination"
+        readOnly={displayWidget}
+        type="text"
+        value={destination}
+      />
+      <button disabled={!displayButtonEnabled} onClick={() => setDisplayWidget(true)}>
+        Display Meeting Widget
+      </button>
+      <button disabled={!displayWidget} onClick={() => setDisplayWidget(false)}>
+        Remove Meeting Widget
+      </button>
+      {token && destination && displayWidget && (
+        <WebexMeetingWidget accessToken={token} meetingDestination={destination} />
+      )}
+    </div>
+  );
+}
+
+WebexMeetingWidgetDemo.propTypes = {
+  token: PropTypes.string,
+};
+
+WebexMeetingWidgetDemo.defaultProps = {
+  token: '',
+};
+
+export default WebexMeetingWidgetDemo;
```

Comment:
All exports are done in the declaration, any reason for this to be done separately?